addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/handler/GardenaThingHandler.java

```diff
@@ -248,6 +269,42 @@ protected void updateStatus(Device device) {
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void handleConfigurationUpdate(Map<String, Object> configurationParameters)
+            throws ConfigValidationException {
+        validateConfigurationParameters(configurationParameters);
+
+        Device device = null;
+        try {
+            GardenaSmart gardena = getGardenaSmart();
+            device = gardena.getDevice(UidUtils.getGardenaDeviceId(getThing()));
+
+            for (Entry<String, Object> configurationParmeter : configurationParameters.entrySet()) {
+                String key = configurationParmeter.getKey();
+                Object newValue = configurationParmeter.getValue();
+                if (newValue != null && SETTING_TURN_ON_PRESSURE.equals(key)) {
+                    newValue = new Double((String) newValue);
+                }
+
+                try {
+                    Setting setting = device.getSetting(key);
+                    if (ObjectUtils.notEqual(setting.getValue(), newValue)) {
+                        gardena.sendSetting(setting, newValue);
+                        setting.setValue(newValue);
+                    }
+                } catch (GardenaException ex) {
+                    logger.error("Error setting thing property {}: {}", key, ex.getMessage());
```

Comment:
In [E.4 of the coding guidelines](http://docs.openhab.org/developers/development/guidelines.html#e-logging) we listed what kind of logging level should be used when. Error should only be used:

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. 

Please change this to use the correct logging levels and if possible change the ONLINE/OFFLINE status of thing / bridge.